src/context.rs

```diff
@@ -1097,23 +1097,29 @@ impl BlockOffset {
 pub struct Block {
   pub mode: PredictionMode,
   pub bsize: BlockSize,
+  pub sb_type: BlockSize,
   pub partition: PartitionType,
   pub skip: bool,
   pub ref_frames: [usize; 2],
   pub neighbors_ref_counts: [usize; TOTAL_REFS_PER_FRAME],
-  pub cdef_index: u8
+  pub cdef_index: u8,
+  pub n8_w: usize, /* block width in the unit of mode_info */
+  pub n8_h: usize, /* block height in the unit of mode_info */
 }
 
 impl Block {
   pub fn default() -> Block {
     Block {
       mode: PredictionMode::DC_PRED,
       bsize: BlockSize::BLOCK_64X64,
+      sb_type: BlockSize::BLOCK_64X64,
```

Comment:
Copying the comment I've left at atomnuker's branch.
"The existing 'bsize' is exactly the same to newly added sb_type."

Reply:
Yes. Both have been removed at this time since they are not used.